packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -23,7 +23,7 @@ export function CellContent({ isTitle, item, column, titleProps, id }) {
 			hideLabel
 		/>);
 	}
-	const str = item[column.key] ? item[column.key].toString() : '';
+	const str = item[column.key] !== undefined ? item[column.key].toString() : '';
```

Comment:
`!=` instead of `!==` will return `false` whenever the value is `null` or `undefined` ; which is more appropriate in this case.